surefire-its/src/test/resources/surefire-2006-shadefire-transformers/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.maven.plugins.surefire</groupId>
+  <artifactId>surefire-2006</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>shadefire usage test</name>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Here we can open more alternatives for test errors if we build the sources against `${java.specificaation.version}`.

Reply:
What is your proposition?
Most of IT tests have the same, even with 1.7

Reply:
The old tests with j7 are old tests, that's the history.
Once, Robert Scholte provided an integration test with
```
<maven.compiler.source>${java.specificaation.version}</maven.compiler.source>
<maven.compiler.target>${java.specificaation.version}</maven.compiler.target>
```
It was very interesting configuration. We know that the bytecode is not always the same beween versions which is good for us because we can find more bug in these tests.